src/common/ui/code.vue

```diff
@@ -50,9 +50,9 @@
 </template>
 
 <script>
+import '#/common/ui/code_extras/mode/javascript-mixed/javascript-mixed';
```

Comment:
`code_extras` is using the wrong convention, it should be `-`, but I'd like to simplify the path e.g. maybe `#/common/ui/code/codemirror-js-mixed`

Reply:
done

Reply:
i just noticed in the same file, `import #/` lines are put at the end of the imports. Should they be moved to the top as well? 

https://github.com/violentmonkey/violentmonkey/blob/033282f44eccc5dd5cf8749704b4f56323eb8e0e/src/common/ui/code.vue#L69-L73

I poked around a few other `.vue` files. There do not appear to have a convention of placing `import #/` at the top of imports.

Reply:
I've just checked and I see `#` being grouped at the top right after `vue` import in every file except one. I guess code.vue treats CodeMirror block as a framework which is why it's at the top. Well, I guess then let's move it at the end of the import block.